pkg/koordlet/koordlet.go

```diff
@@ -22,7 +22,6 @@ import (
 	"os"
 	"time"
 
-	topologyclientset "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/clientset/versioned"
```

Comment:
actually, I don't see any typos here.

Reply:
yes, actually it's not  typos,   thinking "github.com" imports should be gathered together ?

Reply:
we usually follow the convention below:

```
import (
  // stdlib
  "fmt"
  "os"

  // third party packages
  "go.uber.org/atomic"
  "golang.org/x/sync/errgroup"

  // project packages
  "github.com/user/project"
)
```